### ISSUE TITLE
Is it possible to expose GUI (e.g. rstudio) from a Docker image?

### ISSUE THREAD

**Issue by ofrei** [comorment/gwas#9]
This is somewhat similar to issue #6  - but with standard graphical user interface.
For example, can we package an rstudio into a docker or singularity? 
Perhaps this is not possible - I just want to double-check.

**Comment by ofrei** [comment]
@bayramakdeniz Very impressive to see an interactive Octave GUI enabled by a singularity container. Could you check how this works on TSD? Could you make it work also for rstudio?

**Comment by bayramakdeniz** [comment]
For Octave GUI, it seems that X11 DISPLAY environment variable is required to run an interactive GUI. I have tested it on both TSD and OS X and could not get the interactive GUI as we experienced on Linux. But I guess it can be possible to activate it on TSD. 

FOR Rstudio, I was not able to run it as GUI yet (it asks me a  Qt platform for that) but a singularity container to run the studio via browser (server) has been already designed: https://pawseysc.github.io/singularity-containers/23-web-rstudio/index.html

https://www.rocker-project.org/use/singularity/
https://ropenscilabs.github.io/r-docker-tutorial/02-Launching-Docker.html
